volunteers/templates/static/promo.html

```diff
@@ -3,38 +3,38 @@
 <div id="intro">
     <h2 class="intro"><span>FOSDEM</span><br/>Volunteering</h2>
 
-    <p class="intro">This is a site that brings all the volunteers together.
-    Find all the cool tasks that you are interested in.</p>
+    <p class="intro">This site brings all volunteers together.
+    Find cool tasks you are interested in.</p>
 
-    <p id="xtra-info">Problems? For bugs you can report an issue at
-    <a href="https://github.com/FOSDEM/volunteers/issues">Github</a>.</p>
+    <p id="xtra-info">Problems? Fix or report bugs as issues at
+    <a href="https://github.com/FOSDEM/volunteers/issues">GitHub</a>.</p>
 </div><!-- /intro-->
 
 <ul class="features">
     <li><h3>FOSDEM</h3>
-        <p><a href="https://www.fosdem.org">FOSDEM</a> is a free event that offers open source
+        <p><a href="https://www.fosdem.org">FOSDEM</a> is a gratis event that offers libre software
```

Comment:
Hi! Thanks so much for the patch, we very rarely review this text so it's great to have another pair of eyes on it. Could we just use the language from https://fosdem.org ?

> FOSDEM is a free event for software developers to meet, share ideas and collaborate.

> Every year, thousands of developers of free and open source software from all over the world gather at the event in Brussels.

Reply:
Rather change it in in both places, seeing as that isn't how one writes "Open Source", and the "free" of "free software" is invoked in a different meaning (being gratis) in the sentence just before that.

Reply:
Oh gosh, changing the front page 😅 . I could bring it up with the staff mailing list but people are pretty busy 2 weeks before the conf and that sentence has likely been there for who knows how many years, so there might be a pretty significant delay and it's not guaranteed to pass.

Pre-empting one argument which will undoubtedly arise, do you think all the international visitors will understand gratis and libre? Gratis maybe, though it's a bit quaint, but libre is used really rarely nowadays outside core FSF communications. People write FOSS if they explicitly want to include both Free and Open Source, or mostly just write the entire phrase.

Reply:
Well it isn't as if anyone spent any effort right there. I can guarantee the current situation is confusing. FOSS makes less than zero sense to me if it is "free _and_ open …". "Libre" is untainted by FSF and OSI communication. I remember the bar door question was "What does FOSDEM stand for?", and there were many blank stares from even seasoned visitors. The connection to the academic term is maybe less important, and a lost cause making for a cool resulting name. Will put in a PR.